docs/api/desktop-capturer.md

```diff
@@ -100,8 +100,29 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
-## Caveats
+## macOS Caveats
 
-`navigator.mediaDevices.getUserMedia` does not work on macOS for audio capture due to a fundamental limitation whereby apps that want to access the system's audio require a [signed kernel extension](https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/KernelExtensions/KernelExtensions.html). Chromium, and by extension Electron, does not provide this.
+### System Audio Capture (macOS 12.3+)
 
-It is possible to circumvent this limitation by capturing system audio with another macOS app like Soundflower and passing it through a virtual audio input device. This virtual device can then be queried with `navigator.mediaDevices.getUserMedia`.
+ On macOS 12.3+, `navigator.mediaDevices.getDisplayMedia` can be used to capture system audio
+ if the [`setDisplayMediaRequestHandler`](./session.md#sessetdisplaymediarequesthandlerhandler-opts) `callback` 
+ is passed a `streams` object that has `audio: 'loopback'`. See the example above.
+
+> [!NOTE]
+> Due to ongoing changes in Chromium, capturing system audio is experimental on macOS 15+ and requires the 
+> following [command line switch](./command-line-switches.md) to be appended. Please note that you may experience bugs.
+>
+> ```js
+> // main.js
+> app.commandLine.appendSwitch('enable-features', 'MacSckSystemAudioLoopbackOverride');
+> ```
+
+### System Audio Capture (before macOS 12.3)
+
+On macOS 12.3 or lower, system audio capture is not supported due to a fundamental limitation whereby 
```

Comment:
```suggestion
Before macOS 12.3, system audio capture is not supported due to a fundamental limitation whereby 
```

Sounds like it works on 12.3.

Reply:
You can try opting in to it by enabling the `MacSckSystemAudioLoopbackOverride` flag, but various downstream codepaths regard ScreenCaptureKit as [too buggy](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/media/capture/screen_capture_kit_device_utils_mac.mm;l=14-20?q=ScreenCaptureKit&ss=chromium%2Fchromium%2Fsrc) to use until 13.2, so I'm not sure it'll work in practice.